### DISCUSSION TITLE
Help🙏 Error: "underlying network changed" when deploying to zksync testnet (zkSync Remix plugin)

### DISCUSSION THREAD

**Discussion by DappScout** [Cyfrin/foundry-full-course-cu#3041] [Q&A] [answered]
Hi everyone!
I'm trying to deploy a contract to zksync sepolia testnet. Contract is compiling fine, by when I try to deploy it, I'm getting this error:

`Deploying contract Bets
Error: "underlying network changed (event=\"changed\", network={\"chainId\":300,\"name\":\"zkSync Sepolia Testnet\"}, detectedNetwork={\"name\":\"sepolia\",\"chainId\":11155111,\"ensAddress\":null}, code=NETWORK_ERROR, version=providers/5.7.2)"`

I trying to deploy with Wallet as Env:
![image](https://github.com/user-attachments/assets/7b85fc1b-fe73-4707-9b45-7557b2cdcb81)

Any idea why?

**Comment by EngrPips** [comment, marked as answer]
Hello @JakobeG301, Did you ensure that your `Metamask` wallet was on `The ZKSync-sepolia` network and not `the sepolia` network?

**Reply by DappScout** [reply]
@EngrPips Thank you, that was the issue.

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. Keep crushing it.

**Comment by 0xh4ty** [comment]
Switching from firefox to google chrome solved the issue for me.

**Reply by EngrPips** [reply]
Awesome, Thanks for the heads-up buddy.

**Comment by bonalarahul** [comment]
I am encountering same problem, I have checked multiple times and could confirm that my metamask is connect in zksync sepolina testnet, why am I still getting this error

**Reply by EngrPips** [reply]
Refresh your Remix page, clear your browser cache then ensure that your `Metamask` is connected to `zksync-sepolia` and not sepolia and retry.

**Reply by bonalarahul** [reply]
I tried this but It didn't work, what could be the issue?

**Reply by EngrPips** [reply]
> I tried this but It didn't work, what could be the issue?

Maybe try another browser. If the issue persists just move on as everything you learnt on Sepolia is sufficient to keep going.
